README.md

```diff
@@ -92,3 +92,83 @@ Options:
+
+```
+npm run fork
+```
+
+Which will grab the latest polkadot and moonbeam binaries, grab the latest snapshot of live state, modify some values (such as validator keys, and inject balances),
+ and run it as a new local network. From here you can perform any interaction as if it was the real state, such as runtime upgrades, interactions on deployed contracts,
+ staking operations etc.
+
+
+If however, you are more interested in intereacting with the contracts in the Moonbeam emulated EVM environment, but not so much parachain staking or XCM, you can run
+the forked network in development mode. This allows for manual sealing of blocks which dramatically reduces execution times of tests (reduction of 12s blocktime into milliseconds).
+
+```
+npx ts-node npx ts-node ./src/tools/run-moonbeam-fork.ts -n moonbeam --dev
+```
+
+### Further Examples
+
+Calling the script directly can be done via: `npx ts-node ./src/tools/run-moonbeam-fork.ts`
+
+The minimal command is: `npx ts-node ./src/tools/run-moonbeam-fork.ts -n moonbeam`
+
+By default the polkadot and moonbeam binaries will be downloaded from github if none found in the `binaries` folder, but if for whatever reason you need to provide your own (e.g. you are on an Apple Silicon chip)
+use the `--polkadot-binary` option to provide the path to the binary to use (or copy them into the folder). If doing this option, make sure the correct binary version is supplied via `--moonbeam-version` and 
+`--polkadot-version` respectively, otherwise the script will still attempt to download the latest.
+
+:information_source: When running the node in manual-seal mode, to create a block you can run the following `curl` command:
+```
+curl --location --request POST 'http://127.0.0.1:9933/' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"engine_createBlock",
+    "params" :[true,true]
+}'
+```
+
+
+The full list of options can be found in the `--help` :
+```
+      --help                      Show help                            [boolean]
+      --version                   Show version number                  [boolean]
+  -n, --network                   Network to retrieve the exported state for
+                                                             [string] [required]
+      --latest                    Will verify if a latest snapshot is available
+                                  and download it     [boolean] [default: false]
+      --reset-to-genesis          Will delete the execution database, setting
+                                  this will restore network back to genesis
+                                  state               [boolean] [default: false]
+      --purge-all                 Will delete ALL files at base path, use with
+                                  caution             [boolean] [default: false]
+      --dev                       Will run the network as a single manual-sealed
```

Comment:
Since we are running it in --dev, shouldn't we allow to pass "--sealing" too so we can select which one we want (I know Alberto and I often use the --sealing 2000

Reply:
Done